collector/collector_test.go

```diff
@@ -0,0 +1,42 @@
+package collector
+
+import (
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+)
+
+type LabelMap map[string]string
```

Comment:
Also, if you do your own type here, it shouldn't be exported, since it's only used in tests in the same package.